### PR TITLE
CP-49350: fix variable naming in Go SDK

### PR DESCRIPTION
**Pull request by duobei**
fix variable naming, such as "uUID", "pIF", "vM"

### CODE REVIEW

File: ocaml/sdk-gen/go/gen_go_helper.ml [hunk 1]
```diff
@@ -41,16 +41,25 @@ let acronyms =
...
-           String.uppercase_ascii s
-       | s ->
-           String.capitalize_ascii s
-       )
-  |> String.concat ""
+
+let snake_to_camel (s : string) : string =
+  split_str s |> List.map amend_acronym |> String.concat ""
+
+let snake_to_camel_for_name_internal (s : string) : string =
```
Comment (minglumlu) [inline ocaml/sdk-gen/go/gen_go_helper.ml:57]: How about `let snake_to_camel ?(internal=false) s`?

Comment (minglumlu) [reply ocaml/sdk-gen/go/gen_go_helper.ml:57]: And please add some unit tests for this change.

Comment (duobei) [reply ocaml/sdk-gen/go/gen_go_helper.ml:57]: > How about `let snake_to_camel ?(internal=false) s`?

Nice.

**Review by minglumlu — APPROVED** [review]

**Review by xueqingz — APPROVED** [review]
Run xenserver-samples locally, no special error found.

**Pull request merged**